modules/virt-creating-storage-pool-pvc-template.adoc

```diff
@@ -46,7 +46,7 @@ spec:
 ----
 <1> The `storagePools` stanza is an array that can contain both basic and PVC template storage pools.
 <2> Specify the storage pool directories under this node path.
-<3> Optional: The `volumeMode` parameter can be either `Block` or `Filesystem` as long as it matches the provisioned volume format. If no value is specified, the default is `Filesystem`. If the `volumeMode` is `Block`, the mounting pod creates an XFS file system on the block volume before mounting it.
+<3> Optional: The `volumeMode` parameter can be either `Block` or `Filesystem` as long as it matches the provisioned volume format. If no value is specified, the default is `Filesystem`. If the `volumeMode` is `Block`, the mounting pod creates an XFS file system on the block volume before mounting it. Editing this sentence triggers a Vale review comment.
```

Comment:
🤖 **[error] RedHat.TermsErrors**: Use 'if' or 'provided that' rather than 'as long as'.